### ISSUE TITLE
User Story: Train the model

### ISSUE THREAD

**Issue by caketaster** [caketaster/mildew-detection#5]
As a **Developer** I can **train the model using the training images** so that **I can predict the status of an image**

- Create  a model using Tensorflow including convolution, dense and pooling layers, with a dropout layer to minimise overfitting
